concourse/scripts/generate-cf-secrets.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+set -x
+
+SCRIPTS_DIR=`dirname $0`
+OUTPUT_FILE=$1
+
+pwgen() {
+   openssl rand -base64 12 | sed 's/+/0/;s/\//0/'
```

Comment:
Instead of having to deal with 2 problems (now that we're using a regex), we could use `-hex` instead of `-base64` in the openssl command. That would ensure we don't have any problematic characters.